Extension/src/LanguageServer/extension.ts

```diff
@@ -252,9 +252,14 @@ export async function activate(): Promise<void> {
         activeDocument = activeEditor.document;
     }
 
-    if (util.extensionContext && new CppSettings().experimentalFeatures) {
-        const tool = vscode.lm.registerTool('cpptools-lmtool-configuration', new CppConfigurationLanguageModelTool());
-        disposables.push(tool);
+    if (util.extensionContext) {
+        // lmTools wasn't stabilized until 1.95, but (as of October 2024)
+        // cpptools can be installed on versions of VS Code as old as 1.67.
```

Comment:
Instead of hardcoding the 1.67 in the comment can you link to https://github.com/microsoft/vscode-cpptools/blob/main/Extension/package.json#L14 ?

Reply:
Or mention engines.vscode in package.json.

Reply:
Otherwise, the engines.vscode could be increased and the comment might be outdated.

Reply:
Thanks, I updated the comment to remove the hardcoded version and link to package.json on GitHub.